features/documents_widget.feature

```diff
@@ -50,7 +50,6 @@ Feature: Adding the document widget to personalisable pages
   Scenario: Adding a "Documents" widget to the my page
     Given I am on the My page personalization page
     # Safeguard to ensure the page is loaded
-    And I should see "My page block"
```

Comment:
The safeguard is actually not needed anymore, as the my page actually loads a new page (as opposed to before where the content was loaded in via ajax